css/device-catalog.css

```diff
@@ -0,0 +1,884 @@
+}
+.btn-basket:active {
+    color: rgba(0, 0, 0, 0.3);
+}
+.btn-compare {
+    outline: none;
+    border: none;
+    background-color: transparent;
+    opacity: 0.5;
+}
+.btn-compare:hover {
+    opacity: 1;
+}
+.btn-compare:active {
+    color: rgba(0, 0, 0, 0.2)
+}
+.section-filter h2 {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+}
+.sorting-style a {
+    font-family: 'Opensans', 'Opensanslight';
+    font-size: 14px;
+    line-height: 18px;
+}
+.content h3 {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+}
+.content p {
+    font-family: 'Gilroylight', Opensans, sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+}
+.content-buttons button:first-child {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+.content-buttons button:last-child {
+    font-family: 'Opensanslight', 'Opensans', sans-serif;
+    font-size: 13px;
+    line-height: 36px;
+}
+.content-pages {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 77px;
+    background-color: #efefef;
+}
+.content-pages ul {
+    display: flex;
+    justify-content: space-between;
+}
+.content-pages li:nth-child(1) {
+    cursor: default;
+    pointer-events: none;
+}
+.content-pages li:nth-child(2),
+.content-pages li:nth-child(3) {
+    opacity: 0.3;
+}
+.content-pages li:nth-child(1):hover,
+.content-pages li:nth-child(2):hover,
+.content-pages li:nth-child(3):hover {
+    opacity: 0.6;
+}
+.content-pages li:nth-child(2):active,
+.content-pages li:nth-child(1):active,
+.content-pages li:nth-child(3):active {
+    opacity: 1;
+}
+.content-pages li:nth-child(2) {
+    margin-left: 31px;
+    margin-right: 29px;
+}
+.content-pages a {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+.page-back,
+.page-forward {
+    padding-left: 29px;
+    padding-right: 26px;
+    padding-top: 23px;
+    padding-bottom: 23px;
+}
+.page-back:hover,
+.page-forward:hover {
+    background-color: #d9d9d9;
+}
+.page-back:active,
+.page-forward:active {
+    color: rgba(0, 0, 0, 0.3);
+}
+
+/* SIDE BAR */
+
+.side-bar {
+    max-width: 467px;
+    padding-left: 200px;
+    padding-right: 68px;
+
+    background-color: #efefef;
+}
+.side-bar h2 {
+    margin-top: 23px;
+    margin-bottom: 23px;
+}
+.side-bar legend {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+}
+.side-bar label {
+    position: relative;
+    padding-left: 43px;
+
+    font-family: 'Opensanslight', 'sans-serif';
+    font-size: 14px;
+    line-height: 40px;
+}
+.filter-price {
```

Comment:
borders doesn't work for fieldsets? I've checked in the playground - they do) use border-top or border-bottom for these black lines

Reply:
I used borders for the legend not for the fieldset. Looks like in mockup

Reply:
that's slick!))